tv_apps/browser/js/browser_db.js

```diff
@@ -99,33 +89,33 @@ var BrowserDB = {
    * @param {Object} data Top sites data
    * @param {Number} frequency
    */
-  populateTopSites: function browserDB_populateTopSites(data, frequency) {
-    var index = frequency;
-
-    data.forEach(function(topSite) {
-      if (!topSite.uri || ! topSite.title) {
-        return;
-      }
-      this.addTopSite(topSite.uri, topSite.title, index);
-      index--;
-      if (topSite.iconUri) {
-        if (topSite.iconUri instanceof Blob) {
-          var reader = new FileReader();
-          reader.onloadend = (function() {
-            topSite.iconUri = reader.result;
-            this.setAndLoadIconForPage(topSite.uri, topSite.iconUri);
-          }.bind(this));
-          reader.onerror = function() {
-            console.error('Unable to read iconUri from blob');
-          };
-          reader.readAsDataURL(topSite.iconUri);
-          return;
-        }
-
-        this.setAndLoadIconForPage(topSite.uri, topSite.iconUri);
-      }
-    }, this);
-  },
+  //populateTopSites: function browserDB_populateTopSites(data, frequency) {
+  //  var index = frequency;
+  //  console.log("### populateTopsite data.iconUri=" + data.iconUri);
```

Comment:
I suggest we preserve these commented lines. We could decide if we would like to bring back the original v2.0 topsite display logic later.